src/dbnode/storage/bootstrap/bootstrapper/fs/source.go

```diff
@@ -479,6 +485,7 @@ func (s *fileSystemSource) markRunResultErrorsAndUnfulfilled(
 
 func (s *fileSystemSource) loadShardReadersDataIntoShardResult(
 	ns namespace.Metadata,
+	accumulator bootstrap.NamespaceDataAccumulator,
 	run runType,
```

Comment:
meganit: maybe keep runType as the first arg for each of these functions? Though probably more headache flipping tests etc around than it's worth

Reply:
Sounds good.